__tests__/integration/core-blockchain/blockchain.test.ts

```diff
@@ -61,7 +61,7 @@ describe("Blockchain", () => {
 
             expect(transactions.length).toBe(transactionsWithoutType2.length);
 
-            expect(transactions).toEqual(transactionsWithoutType2.map(transaction => transaction.serialized));
+            expect(transactions.sort()).toEqual(transactionsWithoutType2.map(transaction => transaction.serialized).sort());
```

Comment:
You can use [.toIncludeAllMembers](https://github.com/jest-community/jest-extended#toincludeallmembersmembers) then you don't need to sort.